src/Mod/Assembly/Gui/ViewProviderAssembly.cpp

```diff
@@ -38,6 +38,8 @@
 #include <Inventor/sensors/SoSensor.h>
 #endif
 
+#include <chrono>
```

Comment:
What to do about this Link warning? @chennes 

> <chrono> is an unapproved C++11 header.  [build/c++11] [5]

Reply:
That warning doesn't apply to our project (it's from Google's internal guidelines). We should adjust our linter settings to turn it off.

Reply:
Thanks